deployment/pod-security-policy.yaml

```diff
@@ -66,9 +66,9 @@ spec:
   - hostPath
   - secret
   hostNetwork: true
-  hostPort:
+  hostPorts:
   - min: 0
-  - max: 65535
+    max: 65535
```

Comment:
Could you add e2e test for this? Currently we run e2e test by installing provider via helm charts.

We can modify [e2e_test_suite](https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/test/e2e/e2e_suite_test.go) to install provider from [deployment](https://github.com/Azure/secrets-store-csi-driver-provider-azure/tree/master/deployment) dir and run entire suite.

Reply:
The e2e tests are being tracked here: https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/196